options/unixfs.go

```diff
@@ -166,6 +181,40 @@ func UnixfsLsOptions(opts ...UnixfsLsOption) (*UnixfsLsSettings, error) {
 	return options, nil
 }
 
+func UnixfsAddMetaOptions(opts ...UnixfsAddMetaOption) (*UnixfsAddMetaSettings, error) {
+	options := &UnixfsAddMetaSettings{
+		Pin:      false,
+		Events:   nil,
+		Silent:   false,
```

Comment:
what about `Overwrite`? It's best to fully define the defaults

Reply:
it is done.